examples/threathunter/threat_intelligence/stix_taxii.py

```diff
@@ -52,7 +52,7 @@ class TaxiiSiteConfig:
     password: str = None
     collections: str = '*'
     start_date: str = '2019-01-01 00:00:00'
-    size_of_request_in_minutes: int = 60
+    size_of_request_in_minutes: int = 1440
```

Comment:
Just a general question: Why are we setting it to 1440?

Reply:
1440 minutes = 24 hours = 1 day

Reply:
I know what 1440 minutes is. My question is why we are setting it to that vs 2 days or 3.

Reply:
I can see that it is in minutes. My question is why 1 day vs 2 or 3.

Reply:
The size of the request is important, because it determines how quickly the script can run, and how much data it's pulling and processing at once. 

Initially, this default value was 60 minutes, but I was always overriding to 1440 minutes via my config file to speed up the polling of my configured TAXII server (I was using otx.alienvault.com). The user collections available to me to poll usually had data added every day, but not every hour. If I used the default value of 60 minutes, I was making 24 calls to download data when just 1 call would have achieved similar/same results.

I found a similar scenario with open.taxiistand.com, with data added usually daily and not hourly. The script took considerably longer to run when I didn't override the default value to 1440 minutes. 

A counter scenario could be a paid TAXII service, which may be updated more frequently than open source, free TAXII servers. In this case, a smaller `request_size_in_minutes` would be appropriate, to reduce the amount of content downloaded with each call. 

I can confirm scenarios where 1440 minutes is most helpful, but only assume a default of 60 minutes would be helpful. Therefore, I chose 1440 minutes